kafka-init/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG KAFKA_VERSION
+ARG SCALA_VERSION
+
+FROM monasca/kafka:${KAFKA_VERSION:-"0.9.0.1"}-${SCALA_VERSION:-"2.11"}
```

Comment:
I'm not sure if it's a big issue or not, but the pr-bot will not be able to parse this `FROM` tag (and I'm not sure there's a reasonable way to parse it given the the bot's resource model). So this image won't get automatic updates from the base kafka image. We can either live with that (really not too bad) or use a plain `FROM monasca/kafka:0.9.0.1-2.11`.

BTW, any reason to use `${KAFKA_VERSION:-"0.9.0.1"}` rather than just `ARG KAFKA_VERSION=0.9.0.1`? Does docker not evaluate them properly in `FROM` or anything like that?

(it might also be good to depend on the explicit container version somehow, e.g. `monasca/kafka:${KAFKA_VERSION}-${SCALA_VERSION}-1.0.2`)